inc/saiswitch.h

```diff
@@ -511,6 +511,9 @@ typedef enum _sai_switch_attr_t
     /* Custom range base value */
     SAI_SWITCH_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
+    /* Enable Broadcom switch shell [bool] (default to 0) */
+    SAI_SWITCH_ATTR_BRCM_SWITCH_SHELL_ENABLE,
```

Comment:
yes! XPliant's shell is "initialize"/"enable" in a very similar fashion. Internal to the SAI adapter we invoke a thread. The user interface to the shell definitely is different, but should be agnostic to SAI.